librz/core/cmd_descs/cmd_descs.yaml

```diff
@@ -250,6 +250,15 @@ commands:
   - name: o
     summary: Open files and handle opened files
     subcommands: cmd_open
+    details:
+      - name: Examples
```

Comment:
It's probably better to put details in the respective file of each command rather than the top file.

Reply:
this one is ok, and i would keep it, but yeah, could you please also add an example for each entry of `o` subcommands? not here but just for each one

Reply:
Yes, I am going work on that next. Thanks.

Reply:
ok. just committed. please check if everything's alright.

Reply:
I still think this should be moved into a subcommands file - this file defines a top level structure, details do not belong there, in my opinion

Reply:
ok no problem, I will remove that.

Reply:
done.